internal/service/credentials/resource_credential_type.go

```diff
@@ -409,6 +424,16 @@ func (r *CredentialTypeResource) Schema(ctx context.Context, req resource.Schema
 					},
 				},
 			},
+
+			"created_at": schema.StringAttribute{
```

Comment:
This (and other `created_at` attributes) will benefit from the `UseStateForUnknown()` plan modifier.  Not applicable to `updated_at`